project-koku/koku
# [COST-4367] Add product_vcpu to aws_line_items_daily

Since we will need to use product_vcpu to do some calculations for AWS HCS, this PR will carry that column through to the `aws_daily_line_items` trino table that we query for HCS. This also includes a trino migration to drop that table so it can be recreated with the additional column.

## Jira Ticket

[COST-4367](https://issues.redhat.com/browse/COST-4367)

## Description

This change will add `product_vcpu` to `aws_line_items_daily`

## Testing

1. Checkout Main (NOT THIS BRANCH)
2. Bring up koku and make and load the test customer data
3. Check in trino for the columns on `aws_line_items_daily` and see that `product_vcpu` is not there
```
SHOW COLUMNS FROM aws_line_items_daily;
```
4. Checkout this branch
5. Run the trino-migration to drop the table
6. Reload the test customer data
7. Check trino again and see that the `product_vcpu` column is now present on the `aws_line_items_daily` table.

## Notes

...
